### PR TITLE
feat: lib app creation

### PR DESCRIPTION
**Pull request by Darkmixt12**
Odio a claudio

### CODE REVIEW

File: libs/vym/frontend/ui/app/src/lib/components/app.component.html [hunk 1]
```diff
@@ -1 +1,2 @@
 <p>app works!</p>
+<vym-footer></vym-footer>
```
Comment (cvandradg) [inline libs/vym/frontend/ui/app/src/lib/components/app.component.html]: you missing a space here, make sure to check lint or npx nx format

**Review by cvandradg — CHANGES_REQUESTED** [review]
check the mentioned comment

**Comment by cvandradg** [comment]
Also, I don't see evidences that the tests are running fine

**Comment by Darkmixt12** [comment]
Screenshot_1

**Review by cvandradg — APPROVED** [review]
LGTM

**Pull request merged**